### PR TITLE
Freeze for production.

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made? 🤔
Freeze H2 prod while developing HFS

## How was this change tested? 🤨

⚡ ⚠ If this change involves consuming from or writing to another service (or shared file system), ***run [integration test create_object_h2_spec.rb](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test manually in [stage|qa] environment, in addition to specs. ⚡

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,16 @@
 
 happy-heron, or H2 (from "Hydrus 2.0"), is a Rails web application enabling users to deposit scholarly content into the SDR. It replaced [Hydrus](https://github.com/sul-dlss/hydrus).
 
+**********************************
+__H2 is currently frozen in production__
+
+* The `production` branch is what is currently deployed in production.
+* Any neccessary production patches should be deployed to this branch and deployed with `cap prod deploy`.
```
Comment (edsu) [inline README.md:13]: s/deployed to this branch/applied to this branch/ ?

File: config/deploy/prod.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
-server 'sul-h2-prod.stanford.edu', user: 'h2', roles: %w[web app db cron]
+server 'xsul-h2-prod.stanford.edu', user: 'h2', roles: %w[web app db cron]
```
Comment (edsu) [inline config/deploy/prod.rb:3]: Is making the host name invalid here serving a purpose?

**Review by edsu — APPROVED** [review]

File: config/deploy/prod.rb (hunk 2 above)
Comment (justinlittman) [reply config/deploy/prod.rb:3]: Oh, yes. I meant to describe with a comment. Fixing.

File: README.md (hunk 1 above)
Comment (justinlittman) [reply README.md]: Fixed.

**Pull request merged**